pkg/driver/nodeServer.go

```diff
@@ -41,13 +41,18 @@ type CreateMappingParams struct {
     HostGroup   string
 }
 
-type CreateTargetTgParams struct {
-    Address             string
-    Port                string
-    Target              string
-    ISCSITargetPrefix   string
-    TargetGroup         string
-    NumOfLunsPerTarget  int
+type ISCSIVolumeContext struct {
+    Address                     string
+    Port                        string
+    ISCSITarget                 string
+    ISCSITargetPrefix           string
+    TargetGroup                 string
+    HostGroup                   string
+    ChapUser                    string
+    ChapSecret                  string
+    NumOfLunsPerTarget          int
+    UseChapAuth                 bool
+    DynamicTargetLunAllocation  bool
 }
```

Comment:
Are these changes part of the fixes for ES-525. Please only make the changes related to this ticket.